WallPaperDev/WallPaperDev/Networking/NetworkServiceProtocol.swift

```diff
@@ -0,0 +1,12 @@
+//
+// Created by Jackson Ho on 3/17/20.
+// Copyright (c) 2020 Stephen Ouyang. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+protocol NetworkServiceProtocol {
```

Comment:
Thoughts on renaming to `UnsplashAPIService`

Reply:
The rational being that we may have different protocols for each service instead of a single network service

Reply:
I think I'm going to move this protocol to the `UnsplashEndpoint` file